packages/mail/src/classes/mail-service.js

```diff
@@ -40,6 +41,17 @@ class MailService {
     return this;
   }
 
+  /**
+   * Reuse connections
+   */
+  setKeepAlive(keepAlive) {
+    if (keepAlive === false) {
+      return;
```

Comment:
What about turning off keep alive? If this just returns, you can never "unset" it.

Reply:
`keepAlive` is already false by default, its an opt-in feature